pkgs/servers/authelia/default.nix

```diff
@@ -0,0 +1,81 @@
+{ lib, fetchFromGitHub, buildGoModule, installShellFiles, fetchurl }:
+
+let
+  version = "4.36.5";
+  webUiStatic = fetchurl {
```

Comment:
I would move `webUiStatic` in the derivation so that end users can override it using `finalAttrs` and maybe put it as a `passthru` derivation too.

Reply:
good point

Reply:
can you give an example of how `finalAttrs` works?

LMK if the changes will work for that

Reply:
```
mkDerivation (finalAttrs: {
 ui = stuff;
 passthru = { inherit (finalAttrs) ui; }
}
```

enable users to do `pkgs.authelia.overrideAttrs (old: { ui = newUi; }).ui == newUi`

Reply:
It also remove the need for `rec`

Reply:
but the `postPatch` would still need changing no? I'd have thought it'd use the old `ui` value

I guess I can cahnge the postPatch bit to use the bash var `$ui` rather than using the value directly from nix eval `${ui}` ?

Reply:
function-style `mkDerivation` is really `rec` + "magic override support" (as long as you use `finalAttrs`)